etc/discovery-client/discovery-client.yaml

```diff
@@ -4,6 +4,7 @@ internalDir: /etc/discovery-client/internal/
 reconnectInterval: 5s
 logPagePaginationEnabled: false
 maxIOQueues: 0
+kato: 10
```

Comment:
Why 10 and not 15 which was the effective default before the kernel change?

Reply:
would be nice to understand from name or value what this number actually means (seconds?)

and question, is there a defined upgrade process for the discovery client? (asking to understand if we actually update the yaml and not just the executable.. (although you did give it a default value in any case))

Reply:
@mike-lb it can be 15 , it's not critical.
@arturm-lb current RPM implementation won't override configuration so exists system when upgrading it will fall into default value